chef/cookbooks/hyperv/recipes/openstack_install_nova.rb

```diff
@@ -24,3 +24,14 @@
   EOH
   not_if { ::File.exists?("#{node[:openstack][:nova][:installed]}") }
 end
+
+ruby_block "ensure correct python path in shebang" do
```

Comment:
@jdsn it could be useful to have a function to search and replace a line added to core, so if a card will be created, i can work on it. It would have an effort of 2 maximum.